src/content/release-notes.mdx

```diff
@@ -22,7 +22,7 @@ This version is compatible with Kubernetes versions 1.27 to 1.30
 
 This version is compatible with Kubernetes versions 1.27 to 1.30
 
-### Breaking Changes
+### Breaking Changes - 1.26
```

Comment:
Instead of this, you can keep the title as it is, and customize the anchor link like we do [here](https://github.com/okteto/docs/blob/main/src/content/reference/okteto-manifest.mdx#deploy-remotely-recommended-deploy-remotely). I think keeping the title is cleaner

Reply:
Amazing. I didn't realize you could do that.